package.json

```diff
@@ -36,6 +36,7 @@
     "url": "https://github.com/css-modules/icss-utils/issues"
   },
   "homepage": "https://github.com/css-modules/icss-utils#readme",
+  "prettier": {},
```

Comment:
Let's remove it from here

Reply:
i added it because without it prettier will automatically search for a prettierrc and if you have one in your home directory there is no way to format the code locally correctly. This enforces that the default config is used, instead of the first one it finds

Reply:
I think it is IDE problem, we should not fix IDE problems here

Reply:
I found this behavior is wrong, you can't use 0CJS configuration prettier in this case, plugin should not look above root of projects without additional options

Reply:
ah ok, still might be worth leaving for contributors, its not simple to disable this behavior in vscode

Reply:
removed for now 👍